sim/shaman/shocks.go

```diff
@@ -104,7 +104,6 @@ func (shaman *Shaman) registerFlameShockSpell(shockTimer *core.Timer) {
 				shaman.LavaBurst.BonusCritRating -= 100 * core.CritRatingPerCritChance
 			},
 		}),
-		// TODO: is this bonus ticks or bonus time that results in extra ticks?
```

Comment:
(Base duration is 18s, and is shortended by haste. The set adds 9s, and is almost definitely converted into ticks the same way)